lib/helpers/CloudFirebaseHelper.dart

```diff
@@ -0,0 +1,54 @@
+//import 'ItemsHelper.dart';
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:trovami/model/DocItem.dart';
+
+class CloudFirebaseHelper {
+  Future addItemToCloud(DocItem item, String collectionName) async{
+    CollectionReference collection = FirebaseFirestore.instance.collection(collectionName);
+      return collection
+          .add(item.toJson())
+          .then((value) => {
+            print("Item ${item.getName()} (${item.id}) added to cloud"),
+            item.id = value.id,
+            value.update(item.toJson()),
+            print("Item ${item.getName()} (${item.id}) updated with id")
+            })
+          .catchError((error) => print("Failed to add ${item.getName()}: $error"));
+  }
+
+  Future deleteItem(DocItem item, String collectionName) async {
+    CollectionReference collection = FirebaseFirestore.instance.collection(collectionName);
+
+    collection.doc(item.id)
+        .delete()
+        .then((value) => print("FirebaseHelper.deleteItem(): ${item.getName()} deleted"))
+        .catchError((error) => print("Failed to delete ${item.getName()}: $error"));
+  }
+
+  Future<Map<String, Object>> fetchDocs(String collectionName, DocItem item) async{
```

Comment:
"getItem" ?

Reply:
I went with getItems for now.  What would the function be to retrieve from local storage?
If we need to "Get" a particular User or Group from local storage, seeing "getItem" would be unclear if we are hitting the online database or retrieving it locally.  Of course the Class being used would document the source but sometimes the class is not included in the call.

Reply:
Yeah, I don't mind the current name either.